src/cypress/integration/pages/TradePage.js

```diff
@@ -0,0 +1,79 @@
+class TradePage {
+  getSymbol(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol.replace('/', '') + '"]')
+  }
+  enterNotional(symbol) {
```

Comment:
`enterNotional` is misleading. This function is returning the notional input of a tile.

Reply:
fixed in latest commit